### PR TITLE
make pledge-related notifications more specific and helpful

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

File: server/boot/commit.js [hunk 1]
```diff
@@ -217,11 +220,14 @@ export default function commit(app) {
       })
       .subscribe(
         pledge => {
-          let msg = 'You have successfully stopped your pledge.';
+          let msg = `You have successfully stopped your pledge. Please remember
```
Comment (BerkeleyTrue) [inline server/boot/commit.js]: you'll need to use `dedent` here

File: server/boot/commit.js [hunk 2]
```diff
@@ -217,11 +220,14 @@ export default function commit(app) {
       })
       .subscribe(
         pledge => {
-          let msg = 'You have successfully stopped your pledge.';
+          let msg = `You have successfully stopped your pledge. Please remember 
+                      to cancel your recurring donation directly with 
+                      the nonprofit directly if you haven't already done so.`;
           if (!pledge) {
-            msg = `No pledge found for user ${user.username}.`;
+            msg = `It doesn't look like you had an active pledge, so there's
```
Comment (BerkeleyTrue) [inline server/boot/commit.js]: and here

File: server/boot/commit.js [hunk 3]
```diff
@@ -217,11 +220,14 @@ export default function commit(app) {
       })
       .subscribe(
         pledge => {
-          let msg = 'You have successfully stopped your pledge.';
+          let msg = `You have successfully stopped your pledge. Please remember 
+                      to cancel your recurring donation directly with
```
Comment (atjonathan) [inline server/boot/commit.js]: ~~Please remember to cancel your recurring donation directly with the nonprofit directly if you haven't already done so.~~ Please remember to cancel your recurring donation directly with the nonprofit if you haven't already done so.

> Duplicate use of the word **directly**

**Comment by QuincyLarson** [comment]
@BerkeleyTrue @atjonathan I've updated the PR please QA and merge.

File: server/boot/commit.js [hunk 4]
```diff
@@ -217,11 +220,18 @@ export default function commit(app) {
       })
       .subscribe(
         pledge => {
-          let msg = 'You have successfully stopped your pledge.';
+          let msg = dedent`
+                      You have successfully stopped your pledge. Please 
+                      rememberto cancel your recurring donation directly
```
Comment (atjonathan) [inline server/boot/commit.js]: You appear to have some spacing issues here...

File: server/boot/commit.js [hunk 5]
```diff
@@ -217,11 +220,18 @@ export default function commit(app) {
...
+          let msg = dedent`
+                      You have successfully stopped your pledge. Please 
+                      rememberto cancel your recurring donation directly  
+                      with the nonprofit if you haven't already done so.
+                    `;
           if (!pledge) {
-            msg = `No pledge found for user ${user.username}.`;
+            msg = dedent`
+                      It doesn't look like you had an active pledge, so 
+                      there's no pledge to stop.
```
Comment (atjonathan) [inline server/boot/commit.js]: You appear to have some spacing issues here...

Comment (QuincyLarson) [inline server/boot/commit.js]: Fixed. Please merge ASAP.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks @QuincyLarson

**Pull request merged**